### PR TITLE
[Feature] Add portal page

### PR DESCRIPTION
**Pull request by MadRex2000**
(Thanks for sending a pull request! Please fill in the following content to let us know better about this change.)

## Types of changes
Please put an `x` in the box that applies

- [ ] **Bugfix**
- [x] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description
**Describe what the change is**
add portal in index page
solved issue #876 

## Steps to Test This Pull Request
Go to `localhost:8000/<lang>`  by your web browser

### CODE REVIEW

**Comment by thicyo** [comment]
screencapture-staging-pycon-tw-2020-zh-hant-2020-08-16-16_51_01


```css
.portal .card {
    display: flex;
    margin-bottom: 15px;
    width: 156px;
    height: 50px;
    justify-content: center;
    align-items: center;
    border-radius: 4px;
    border: solid 2px #6a3f3b; 
}

.portal .card:hover {
    border: solid 2px #e23728; 
}

.portal .card a {
    display: flex;
    width: 100%;
    height: 100%;
    align-items: center;
    justify-content: center;
    /* color: #502526; */ 
    font-size: 16px;
    font-weight: normal;
    letter-spacing: -0.08px;
}
```

File: src/static/pycontw-2020/styles/index.scss [hunk 1]
```diff
@@ -29,26 +29,27 @@ body {
...
 		display: flex;
+    margin: 0 5px;
 		margin-bottom: 15px;
 		width: 156px;
 		height: 50px;
 		justify-content: center;
 		align-items: center;
 		border-radius: 4px;
-	  border: solid 2px #224398;
+	  border: solid 2px #6A3F3B;
```
Comment (uranusjr) [inline src/static/pycontw-2020/styles/index.scss:39]: 這縮排⋯⋯

File: src/static/pycontw-2020/styles/index.scss [hunk 2]
```diff
@@ -29,26 +29,27 @@ body {
 
 	.card {
 		display: flex;
+    margin: 0 5px;
```
Comment (uranusjr) [inline src/static/pycontw-2020/styles/index.scss:32]: 為什麼會差這麼多

**Review by yychen — APPROVED** [review]
Go go!

**Pull request merged**